runtime/dolphin/src/lib.rs

```diff
@@ -246,8 +255,10 @@ impl Contains<Call> for BaseFilter {
 				| manta_collator_selection::Call::remove_collator{..}
 				| manta_collator_selection::Call::leave_intent{..})
 			| Call::Balances(_)
-			| Call::XTokens(_)
-			| Call::Utility(_) => true,
+			| Call::Treasury(_)
```

Comment:
Is enablign the treasury calls on purpose ? If so can we add it to the description

Reply:
Will fix that. It turns out not necessary.